src/lib/controllers/ui-controller.js

```diff
@@ -1024,8 +1024,9 @@ export default class UIController {
 
   updateVerboseMode () {
     this.state.setItem('verboseMode', this.contentOptions.items.verboseMode.currentValue === this.options.verboseMode)
-    this.panel.panelData.verboseMode = this.options.verboseMode
-    this.popup.popupData.verboseMode = this.options.verboseMode
+
+    this.panel.panelData.verboseMode = (this.contentOptions.items.verboseMode.currentValue === this.options.verboseMode)
```

Comment:
could also just use the state variable here, after setting it.

Reply:
I made as less changes here as possible, I know, that @kirlat is working on state handling and options. So to avoid much mix working on the same I have changed it very simple.

Reply:
ok that's fine.